src/frontend/screens/Game/GamePage/components/InstalledInfo.tsx

```diff
@@ -111,10 +110,10 @@ const InstalledInfo = ({ gameInfo }: Props) => {
           <div>
             <b>Wine:</b> {wineName}
           </div>
-          {wineVersion && wineType === 'crossover' ? (
+          {wineType === 'crossover' ? (
             <div>
               <b>{t2('setting.winecrossoverbottle', 'Bottle')}:</b>{' '}
-              <div className="truncatedPath">{winePrefix}</div>
+              <div>{wineCrossoverBottle}</div>
```

Comment:
This isn't really related to this PR, but I think we should do these types of translations a little differently. Not every language will follow the "Label: Value" format